src/Symfony/Installer/DemoCommand.php

```diff
@@ -61,7 +62,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 ->checkSymfonyRequirements()
                 ->displayInstallationResult()
             ;
+        } catch (AbortException $e) {
+            aborted:
+
+            $output->writeln('');
+            $output->writeln('<error>You aborted the command. We clean up things for you.</>');
+
+            $this->cleanUp();
+
+            return 1;
         } catch (\Exception $e) {
+            // Guzzle can wrap the AbortException in a GuzzleException
+            if ($e->getPrevious() instanceof AbortException) {
+                goto aborted;
```

Comment:
But it increase code complexity, because the `cleanUp` is protected. So it will need an abstract method in the parent class. Does it worth it ? ;)